src/parser/core/modules/Abilities.js

```diff
@@ -78,6 +78,19 @@ class Abilities extends Module {
     return ability;
   }
 
+  /**
+   * Returns the spell with the given spellId from an ability (or undefined if there is no such spell)
+   */
+  getAbilitySpell(spellId) {
```

Comment:
What my idea was here was to group all the rampage damage spells into the one ability. Each hit from rampage has a different damage coefficient, and as I need this in the analyzer to calculate the effective damage from the trait, I needed to get the value from the spell (the damage event doesn't have the full spell details and so is missing the damage coefficient I added in)

I thought of having it as a constant in the analyzer itself, but thought this would be a bit neater and possibly useful in the future.